libs/geospatial/src/main/java/org/codice/ddf/libs/geo/util/GeospatialUtil.java

```diff
@@ -171,4 +196,73 @@ public static Double parseDMSLongitudeWithDecimalSeconds(String dmsLon)
 
         return lon;
     }
+
+    /**
+     * Transform a geometry to EPSG:4326 format with lon/lat coordinate ordering.
+     * NOTE: This method will perform the transform swapping coordinates even if the sourceCrsName is
+     * EPSG:4326
+     * @param geometry - Geometry to transform
+     * @param sourceCrsName - Source geometry's coordinate reference system
+     * @return Geometry - Transformed geometry into EPSG:4326 lon/lat coordinate system
+     */
+    public static Geometry transformToEPSG4326LonLatFormat(Geometry geometry, String sourceCrsName) {
+        try {
+            CoordinateReferenceSystem sourceCrs = CRS.decode(sourceCrsName);
+            Hints hints = new Hints(Hints.FORCE_LONGITUDE_FIRST_AXIS_ORDER, Boolean.TRUE);
+            CRSAuthorityFactory factory =
+                    ReferencingFactoryFinder.getCRSAuthorityFactory("EPSG", hints);
+            CoordinateReferenceSystem targetCRS = factory.createCoordinateReferenceSystem(
+                    EPSG_4326);
+            MathTransform transform = CRS.findMathTransform(sourceCrs, targetCRS);
+            return JTS.transform(geometry, transform);
+        } catch (FactoryException | TransformException e) {
+            LOGGER.debug("Unable to convert {} into {}", sourceCrsName, EPSG_4326, e);
+        }
+        return null;
```

Comment:
Should we throw instead of returning null? how would a caller know what went wrong without the exception?